pkg/sql/alter_index_visible.go

```diff
@@ -28,9 +33,49 @@ type alterIndexVisibleNode struct {
 func (p *planner) AlterIndexVisible(
 	ctx context.Context, n *tree.AlterIndexVisible,
 ) (planNode, error) {
-	return nil, unimplemented.Newf(
-		"Not Visible Index",
-		"altering an index to visible or not visible is not supported yet")
+	if err := checkSchemaChangeEnabled(
+		ctx,
+		p.ExecCfg(),
+		"ALTER INDEX VISIBILITY",
```

Comment:
nit: define a constant

Relates to https://github.com/cockroachdb/cockroach/pull/85473#discussion_r944703867